cli/cluster/file.go

```diff
@@ -0,0 +1,32 @@
+package cluster
+
+import (
+	"fmt"
+	"os"
+)
+
+// FileCollector collects data from a YAML file.
+type FileCollector struct {
+	path string
+}
+
+// NewFileCollector create a new file collector for a path.
+func NewFileCollector(path string) Collector {
```

Comment:
Do not return interfaces, return concrete implementation.

Reply:
Returns `FileCollector` at now.